packages/react/package.json

```diff
@@ -43,10 +48,16 @@
     "tailwind-merge": "^1.13.2"
   },
   "peerDependencies": {
+    "next": "*",
     "react": "^17.0.2 || ^18.0.0",
     "react-dropzone": "^14.2.3",
     "uploadthing": "^5.0.0"
   },
+  "peerDependenciesMeta": {
+    "next": {
+      "optional": true
+    }
+  },
```

Comment:
needed so that tsup treats next as an external, but we don't want to require next.js

Reply:
Will this make all React users require Next though? 😅